### ISSUE TITLE
[NetRocks] FTP client fails to work with uftpd

### ISSUE THREAD

**Issue by unxed** [elfmz/far2l#1067]
https://github.com/troglobit/uftpd

Try opening any folder on FTP server, or downloading any file

**Comment by unxed** [comment]
Disabling «Use MSLD/MLST if possible» fixes it

**Comment by elfmz** [comment]
Looks there is bug in that server, see some test sessions from netcat (NC):
```
$nc GOOD_SERVER 21
....
user anonymous
230 Anonymous user logged in
FEAT
211-Extensions supported:
 EPRT
 IDLE
 MDTM
 SIZE
 REST STREAM
 MLST type*;size*;sizd*;modify*;UNIX.mode*;UNIX.uid*;UNIX.gid*;unique*;
 MLSD
 ESTP
 PASV
 EPSV
 SPSV
 ESTA
 AUTH TLS
 PBSZ
 PROT
 UTF8
211 End.
MLST .
250-Begin
 type=cdir;sizd=1024;modify=20210505090033;UNIX.mode=0755;UNIX.uid=0;UNIX.gid=0;unique=804gde42; .
250 End.
```

And here is uftpd:
```
$ nc 192.168.1.46 2222
220 uftpd (2.13) ready.
user anonymous
230 Guest login OK, access restrictions apply.
FEAT
211-Features:
 EPSV
 PASV
 SIZE
 UTF8
 REST STREAM
 MLST modify*;perm*;size*;type*;
211 End
MLST .
```
...and it stucks forever without any responce. At server log there're messages:
```
244124> Recv: MLST .
244124> Compose path from cwd: /, arg: .
244124> Server path from CWD: /.
244124> Resulting non-chroot path: /home/user/projects/git/uftpd/.
244124> Final path to file: /home/user/projects/git/uftpd
244124> Reading directory /home/user/projects/git/uftpd ... 28 number of entries
```
Summarizing -  server indicates that it supports MLST by FEAT command, but when attempting to use it - server stucks. Although from logs - it tries to do something meaningful, but stucks as result..

**Comment by troglobit** [comment]
Should be fixed with today's [v2.14 release](https://github.com/troglobit/uftpd/releases/tag/v2.14) of uftpd.
